escodegen.js

```diff
@@ -2434,7 +2434,15 @@
 
         ModuleSpecifier: function (expr, precedence, flags) {
             return this.Literal(expr, precedence, flags);
-        }
+        },
+
+        ImportExpression: function(expr, precedence, flag) {
+            return [
```

Comment:
I think we need to do something like

```js
return parenthesize([
  // ...
], Precedence.Call, precedence);
```

in the same way we do when rendering `CallExpression`s.

Reply:
Ah, right. Did that and added a test with a higher precedence operator in 46f2a5f.